io-engine/src/lvs/lvs_lvol.rs

```diff
@@ -1229,6 +1226,7 @@ impl SnapshotOps for Lvol {
                 .map(|b| Lvol::try_from(b).unwrap())
                 .collect::<Vec<Lvol>>();
             // return empty snapshot parameter list, if no snapshot found.
+            // 1 count is refer to the source lvol of snapshot
```

Comment:
tempted to suggest this should be removed, since this is just to avoid a single for loop entry AFAICT and may induce errors, case in point

Reply:
Yes.. that correct. by removing this check also We can achieve the same result. Will fix it.